app-center-webapps/src/main/webapp/skin/less/myApplications.less

```diff
@@ -0,0 +1,32 @@
+/*
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2025 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+@import "./variables.less";
+
+#myApplications {
+  .background-grey-primary {
+    background-color: @primaryBackground;
+  }
```

Comment:
In order to optimise the CSS Definition Map, can you please move this into `helpers.less` in `Meeds-io/platform-ui` project in order to allow reuse it